README.md

```diff
@@ -288,17 +286,16 @@ module.exports = {
         loader: "eslint-loader",
         options: {
           outputReport: {
-            filePath: 'checkstyle.xml',
-            formatter: require('eslint/lib/formatters/checkstyle')
+            filePath: "checkstyle.xml",
+            formatter: require("eslint/lib/formatters/checkstyle")
           }
         }
-      },
-    ],
-  },
-}
+      }
+    ]
+  }
+};
 ```
 
-
 ## Gotchas
 
 ### NoErrorsPlugin
```

Comment:
Let's avoid changes in `README` in this PR, because it is not related to problem